### PR TITLE
Search results

### PR DESCRIPTION
**Pull request by Donnyd88**

### CODE REVIEW

File: src/pages/OrgLogin.jsx
Comment (dinaolmelak) [inline src/pages/OrgLogin.jsx:1]: The `OrgLogin.jsx` had been removed yesterday.

**Review by dinaolmelak — CHANGES_REQUESTED** [review]

File: src/pages/org-login/OrgLogin.module.scss
Comment (dinaolmelak) [inline src/pages/org-login/OrgLogin.module.scss:1]: Also this `/orgLogin` folder

**Review by dinaolmelak — CHANGES_REQUESTED** [review]

File: src/pages/org-login/OrgLogin.module.scss
Comment (dinaolmelak) [reply src/pages/org-login/OrgLogin.module.scss:1]: I removed it on Kendiya's PR since we are not using that anyway

Comment (Donnyd88) [reply src/pages/org-login/OrgLogin.module.scss:1]: Sounds good. I deleted them on this branch.

Comment (dinaolmelak) [reply src/pages/org-login/OrgLogin.module.scss:1]: Great! Thank you @Donnyd88

**Review by dinaolmelak — APPROVED** [review]

**Pull request merged**